lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -278,11 +308,13 @@ sub _hashref {
 
 sub to_hash {
     my ($job, %args) = @_;
-    my $j = _hashref($job, qw/id name priority state result clone_id retry_avbl t_started t_finished test group_id worker_id/);
+    my $j = _hashref($job, qw/id name priority state result clone_id retry_avbl t_started t_finished group_id worker_id/);
     if ($j->{group_id}) {
         $j->{group} = $job->group->name;
     }
     $j->{settings} = $job->settings_hash;
+    # to be removed - lowercase is schema version 38
```

Comment:
in case you wonder why: these hashes are exported in the list queries - and I don't want to break whatever code checks the attribute. But there is no point in keeping it for long